i18n/appointments/locale_en.json

```diff
@@ -130,6 +130,7 @@
   "PATIENT_LIST_VIEW_SEARCH_HEADER": "Search",
   "NO_APPOINTMENT_LIST_VIEW": "No appointments found",
   "NO_APPOINTMENT_CALENDAR_VIEW": "No appointments found",
+  "NO_APPOINTMENT_SUMMARY_VIEW": "This week, no appointments are available for any of the {gridName}",
```

Comment:
What does `{gridName}` mean? During translation, the translators won't understand this word. Can you rephrase the variable to something meaningful for translators to understand. Is this `daysOrWeeks`? or `serviceName`?

Reply:
Hey @gsluthra, The gridName is a param which is being passed from the GridSummary Component. https://github.com/Bahmni/openmrs-module-appointments-frontend/pull/291/files#diff-de2a9964b0394faa6785c534604c44ddb74e68bfddcb4e0e317ca850cc9f0c94R108
`{intl.formatMessage({id: "NO_APPOINTMENT_SUMMARY_VIEW"}, {gridName: gridName})}`

 The param gridName is already translated and sent to the GridSummary so as is would be replaced.
`const specialitiesTitle = <FormattedMessage id={'SPECIALITIES'} defaultMessage={'Specialities'}/>`
`<GridSummary gridData={locationData} weekStartDate={startDate} onClick={goToListView} gridName={locationsTitle}/>`

Reply:
Then this should be: `This week, no appointments are available for any of the {locationName}`  
The reason I say this is because depending on the context of the word in { }, the translators will decide how to write this message. If its locationName, serviceName, Dates, etc... the sentence structure might change.

Reply:
@gsluthra GridSummary is a generic component and we are accepting gridName and gridData as props, so the gridName is already translated based on the locale and we have also added a default message.

Reply:
I am not convinced. Can you please share example of what final messages might looks like, so i can decide if this makes sense. Language translators need to know some context of the message, and gridName is technical and doesn't tell people what the sentence eventually is going to say.

Reply:
Please see section titled "Word Order" in this article to understand what I mean: https://ux.shopify.com/how-to-concatenate-localized-strings-mindfully-bcc7b6d4961a

Reply:
Hi Gurpreet, Based on the gridName the message would be like
Incase of Specialities :
This week, no appointments are available for any of the Specialities
For Providers:
This week, no appointments are available for any of the Providers

Screenshot for Reference:
<img width="1792" alt="Screenshot 2023-12-14 at 10 09 01 AM" src="https://github.com/Bahmni/openmrs-module-appointments-frontend/assets/126503818/3efaca0f-8787-4b0e-aaa1-2c365251077f">

Reply:
In that case the variable can be renamed to `{specialties_or_providers}`

Reply:
The locale strings have been split into individual different strings now by @Phanindra-tw . looks good. thanks.